src/main/java/se/sundsvall/invoicesender/integration/party/PartyIntegration.java

```diff
@@ -24,18 +25,24 @@ public Optional<LegalIdAndPartyId> getPartyId(final String legalId, final String
 		// Strip everything but digits from the legal id
 		var legalIdWithDigitsOnly = legalId.replaceAll("\\D", "");
 
+		var currentYear = LocalDate.now().getYear();
+		// Example legal id : 900101-1234. Birth year is the first two digits of the legal id.
+		var birthYear = Integer.parseInt(legalIdWithDigitsOnly.substring(0, 2));
```

Comment:
I would like to see some tests verifying this. Break the logic out in a utility helper method and make some nice tests cases validating that this works

Reply:
The break out simply to ease with testing.